web/static/js/phoenix.js

```diff
@@ -284,21 +290,48 @@ export class LongPoller {
 }
 
 
-export var Ajax = {
+export class Ajax {
 
-  states: {complete: 4},
+  static request(method, endPoint, accept, body, ontimeout, callback){
+    if(root.XDomainRequest){
```

Comment:
`root` is undefined here (and anywhere else it's used in this file. Please see the phoenix.js on master where we are now referencing this stuff off the global context https://github.com/phoenixframework/phoenix/blob/master/web/static/js/phoenix.js#L292-L293

This PR works fine with WebSocket, but LongPoller fails because of the undefined `root` references.